presto-main/src/test/java/com/facebook/presto/security/TestFileBasedSystemAccessControl.java

```diff
@@ -130,11 +156,11 @@ public void testViewOperations()
         }));
     }
 
-    private AccessControlManager newAccessControlManager(TransactionManager transactionManager)
+    private AccessControlManager newAccessControlManager(TransactionManager transactionManager, String resourceName)
```

Comment:
Please add the test which verifies any user can be used when `user_patterns` section is not present in config file and I think it is good to go.

Reply:
Right. Sorry I missed it earlier. Tell me if something needs to be improved or split into different test functions.